Jenkinsfile

```
@@ -0,0 +1,65 @@
+pipeline {
+  agent { label 'linux' }
+
+  options {
+    disableConcurrentBuilds()
+    /* manage how many builds we keep */
+    checkoutToSubdirectory('src/')
+    buildDiscarder(logRotator(
+      numToKeepStr: '20',
+      daysToKeepStr: '30',
+    ))
+  }
+
+  environment {
+    CONTENT_SOURCE_TYPE  = 'local_folder'
+    CONTENT_SOURCE_URL = 'src/'
+    GIT_COMMITTER_NAME = 'status-im-auto'
+    GIT_COMMITTER_EMAIL = 'auto@status.im'
+    /* dev page settings */
+    DEV_SITE = 'dev.logos.co'
+    DEV_HOST = 'jenkins@node-01.do-ams3.sites.misc.statusim.net'
+    SCP_OPTS = 'StrictHostKeyChecking=no'
+  }
+
+  stages {
+    stage('Clone Builder') {
+      steps {
+        git branch: 'v0', url: 'https://github.com/acid-info/logos-site-builder'
+      }
+    }
+ 
+    stage('Install') {
+      steps {
+        sh 'yarn install'
+      }
+    }
+
+    stage('Build') {
+      steps {
+        sh 'yarn build'
+      }
+    }
+
+    stage('Publish Prod') {
+      when { expression { env.GIT_BRANCH ==~ /.*master/ } }
+      steps {
+        sshagent(credentials: ['status-im-auto-ssh']) {
+          sh "ghp-import -p public"
+        }
+      }
+    }
+
+    stage('Publish Devel') {
+      when { expression { env.GIT_BRANCH ==~ /.*develop/ } }
+      steps {
+        sshagent(credentials: ['jenkins-ssh']) {
+          sh """
+            rsync -e 'ssh -o ${SCP_OPTS}' -r --delete public/. \
+              ${env.DEV_HOST}:/var/www/${env.DEV_SITE}/
+          """
+        }
+      }
+    }
+  }
```

We need some cleanup to not pollute future builds with previous builds:s
```suggestion
  }

  post {
    cleanup { cleanWs() }
  }
```